lib/orch_zmq_config.cpp

```diff
@@ -0,0 +1,34 @@
+#include "orch_zmq_config.h"
+
+std::set<std::string> swss::load_zmq_tables()
+{
+    DBConnector db("CONFIG_DB", 0, false);
+    auto zmq_tables = db.hget(DEVICE_METADATA_LOCALHOST, ZMQ_TABLE_LIST_FIELD);
```

Comment:
This part of code depends on another PR merge first: https://github.com/sonic-net/sonic-buildimage/pull/22451